modules/standard/IO.chpl

```diff
@@ -3857,6 +3857,14 @@ proc channel.writeIt(const x) throws {
      on the type of channel. This operator returns a const reference to the
      same channel so that multiple operator calls can be chained together.
 
+     .. warning:: The <~> operator on channels is deprecated. The intended
+       alternative is to use methods like :proc:`channel.read` and
+       :proc:`channel.write` instead.
+
+       Methods that provide both reading and writing functionality will soon be
```

Comment:
As an end user, this is not clear to me. Does the IO operator provide both reading and writing functionality? Is it a method? Maybe a more general term than "method" would work if you're trying to explain why `<~>` is deprecated.

Reply:
The IO operator does provide reading and writing functionality. My intention with this second paragraph was to point out that this general read-or-write functionality was going away, and so there wouldn't be a method to replace the <~> operator directly.